packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -437,6 +437,13 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
     return channel(mapId).invokeMethod<double>('map#getZoomLevel');
   }
 
+  /// Converts a distance in meters to content size.
+  Future<double> getPoint(double meters, LatLng latLng, {@required int mapId}) {
+    assert(meters != null && latLng != null);
+    return channel(mapId).invokeMethod<double>('map#getPoint',
+        <String, dynamic>{"meter": meters, "location": latLng.toJson()});
+  }
```

Comment:
`getPoint` sounds like a too generic method name for what this does. Can we call this `getPointsForMeters` so it follows the [native API](https://developers.google.com/maps/documentation/ios-sdk/reference/interface_g_m_s_projection#a7275229ff43ac7779c28979855c3c02f) more closely?

Also, the method name over the method channel can be: `projection#getPointsForMeters`? This'd leave the door open to creating a more complete set of `projection` calls.

I'm not an expert in these APIs, so I'm going to defer the final opinion to @cyanglaz!

Reply:
yes, this makes sense. will implement the changes once @cyanglaz confirms.

Reply:
this change has been implemented.